cmake_modules/TokuSetupCompiler.cmake

```diff
@@ -192,14 +199,18 @@ endif ()
 set(CMAKE_C_FLAGS "-Wall -Werror ${CMAKE_C_FLAGS}")
 set(CMAKE_CXX_FLAGS "-Wall -Werror ${CMAKE_CXX_FLAGS}")
 
+## It looks like there is no more need in these changes for at least
```

Comment:
Suggest to remove not comment out (git history will help if it ever needs restoring)